.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: weekly
```

Comment:
Let's make this monthly, please. Following https://github.com/golang/go/issues/21324#issuecomment-1293689721, the expectation is for x/sys to tag ~monthly, which is our only real dependency.

I don't want to follow every quicktest point release, that's unnecessary. Monthly sounds like a good-enough interval to make PRs like https://github.com/cilium/ebpf/pull/866 unnecessary, while staying reasonably up-to-date regarding subtle bugs introduced by newer Go versions. Obviously this is only ever an issue around Go release windows, but having updated dependencies may reduce papercuts for downstream projects anyway.